sphinx/ext/autosummary/__init__.py

```diff
@@ -803,7 +803,7 @@ def process_generate_options(app: Sphinx) -> None:
 
     suffix = get_rst_suffix(app)
     if suffix is None:
-        logger.warning(__('autosummary generats .rst files internally. '
+        logger.warning(__('autosummary generates .rst files internally. '
```

Comment:
For this one, you need to check the localized strings as well otherwise we could have other surprises such as #11689 (though, since it's a logger message it shouldn't affect people that much, but better be safe).

Reply:
My wrong, I thought the localisation files are updated separately by an "out-of-band" process. I have updated all localisation files.